OpenRA.Game/Network/Session.cs

```diff
@@ -26,6 +26,8 @@ public class Session
 		// Keyed by the PlayerReference id that the slot corresponds to
 		public Dictionary<string, Slot> Slots = new Dictionary<string, Slot>();
 
+		public List<int> DisabledSpawns = new List<int>();
```

Comment:
I've been flip-flopping around on names, but with #18672 settled on using "SpawnPoint" for these integer labels. Can we please rename this and the methods that touch it to use "SpawnPoint" too?

Reply:
I have pushed a commit that does this - I hope I have interpreted your wishes correctly :-)